scripts/save_batches.py

```diff
@@ -70,9 +74,9 @@ def _get_datapipe(config_path, start_time, end_time, batch_size, renewable: str
         end_time=end_time,
     )
 
-    data_pipeline = (
-        data_pipeline.batch(batch_size).map(stack_np_examples_into_batch).map(batch_to_tensor)
-    )
+    # data_pipeline = (
```

Comment:
Can we keep this in in some way. Maybe if `renewable == "pv"` or `output_format == "torch"` or something

Reply:
Yep!